e2e/pages/account-settings-page.js

```diff
@@ -0,0 +1,20 @@
+import { UserProfileForm } from "./widgets/account-settings/profile-form";
+import { PasswordForm } from "./widgets/account-settings/password-form";
+import { ProfileHeader } from "./widgets/account-settings/profile-header";
+
+export class AccountSettingsPage {
+  static url = () => "/account";
+
+  static visit = () => {
+    return new AccountSettingsPage().visit();
+  };
+
+  visit() {
+    cy.visit(AccountSettingsPage.url());
+    return this;
+  }
+
+  header = new ProfileHeader();
+  profileForm = new UserProfileForm();
```

Comment:
I'm not sure if the page contains the profile form. I think the page contains tabs, and in a specific tab there is a form.